### ISSUE TITLE
Smarter/dynamic team member layout

### ISSUE THREAD

**Issue by Serneum** [raviriley/agency-jekyll-theme#51]
### **Is your feature request related to a problem? Please describe.**

For a small team, the Team section is left-aligned.

### **Describe the solution you'd like**

The team section uses Boostrap sizing, and uses `col-sm-4`. It would be nice if the theme was a bit smarter, and could:
- Detect the number of team members from the YML
- Determine the best spacing based on the number of team members

For example, for a team of one, it would be nice to see the team member centered. For a team of two, have each block take up half of the width of the screen. For a team of four, one row of three configured with `col-sm-4` like it is now, and a second row where the 4th member is centered.

### **Describe alternatives you've considered**

I've played with the HTML in my browser to change the class from `col-sm-4` to `col-sm-12` to accomplish my goal. Because it is part of a gem, I would need to fork the gem and update it for my one-person team.

![image](https://user-images.githubusercontent.com/1091716/185766669-1e45d18a-4bfa-4595-8160-da8408d2a242.png)

**Comment by raviriley** [comment]
Given that you've played around with it, do you have an idea of the values this should hold?

Psudeocode:
```
# first, make num floor divided by 3 rows of the current spacing 
for (num // 3) rows
  `col-sm-4`
# then, generate a row spaced by if there's 1 or 2 extra team members
extra = num % 3
for (extra) rows
  if extra % 1 == 0
    `col-sm-?`  # value?
  else if extra % 2 == 0
    `col-sm-?`  # value?
```

**Comment by Serneum** [comment]
Given that it's Bootstrap, I'd recommend `col-sm-12` for a row with just one team member so that they are centered, and `col-sm-6` for a row with two team members so that they are each centered in their own half of the screen.

**Comment by raviriley** [comment]
Got it

**Comment by xuzhuoxi** [comment]
I just created a new Github repository, copied all your content into it, and opened Github Pages.
https://github.com/xuzhuoxi/xuzhuoxi2.github.io

**Comment by avicndugu** [comment]
Is anyone working on this feature? I would like to handle it.

**Comment by avicndugu** [comment]
I have resolve this issue. Let me know if I handled the issue fully.

Here are some screenshots of how the team members look.

![Web capture_18-1-2023_191529_jekyll-agency-theme avicndugu repl co](https://user-images.githubusercontent.com/37213663/213240138-d1ecea6b-f1da-4c50-bd47-5d1841629715.jpeg)
![Web capture_18-1-2023_191451_jekyll-agency-theme avicndugu repl co](https://user-images.githubusercontent.com/37213663/213240154-5d431a44-8a11-4987-bf1c-39d90b93ad7a.jpeg)
![Web capture_18-1-2023_191332_jekyll-agency-theme avicndugu repl co](https://user-images.githubusercontent.com/37213663/213240161-048ba477-71fb-4a48-89b6-610a34af84c8.jpeg)
![Web capture_18-1-2023_191248_jekyll-agency-theme avicndugu repl co](https://user-images.githubusercontent.com/37213663/213240168-345e169a-a755-4161-8b53-c5d97f8a549a.jpeg)

**Comment by raviriley** [comment]
Closed by #64
